.github/workflows/my-ci-pipeline.yml

```diff
@@ -15,3 +15,13 @@ jobs:
     -  run: docker build --target test --tag my-test-image .
     -  run: docker run my-test-image
 
+  to-do-app-production:
+    name: Build and Deploy to Azure
+    runs-on: ubuntu-latest
+    needs: to-do-app-tests
+    steps:
+    -  uses: actions/checkout@v2
+    -  run: docker build --target production --tag ${{ secrets.DOCKERHUB_USERNAME }}/todoapp .
+    -  run: docker login --username ${{ secrets.DOCKERHUB_USERNAME }} --password ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
Minor, but you'll see this flags a warning saying:
> WARNING! Using -*** the CLI is insecure. Use --password-stdin.

We should ideally action this step using the `--password-stdin` flag rather than passing the password explicitly, to avoid leaking those credentials to shell logs. [See the docs](https://docs.docker.com/engine/reference/commandline/login/) for more info, example of how that might look below:
```
echo ${{ secrets.DOCKERHUB_TOKEN }} | docker login --username ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
```

Reply:
Thanks Jack. I've made this change and it tested it's all still working. Thank you.